mongodbatlas/data_source_mongodbatlas_privatelink_endpoint_service_serverless_test.go

```diff
@@ -1,118 +0,0 @@
-package mongodbatlas
```

Comment:
I have moved the data source tests into the resource tests. By doing this, there is no need to create a new serverless cluster specifically for these tests. Instead, we can efficiently reuse the clusters that are already created for the resource tests.